AudibleApi/Api.Library.cs

```diff
@@ -163,8 +91,7 @@ public enum ResponseGroupOptions
 			Series = 1 << 22,
 			[Description("sku")]
 			Sku = 1 << 23,
-			// https://stackoverflow.com/questions/7467722
-			ALL_OPTIONS = ~(1 << 24)
+			ALL_OPTIONS = (1 << 24) - 1
```

Comment:
Curious why on these you prefer subtraction over bit inverting

Reply:
> Please test extensively if you have not already including pretending that you're a fresh-install user.
I've tested login as well as everything else in libation that touches the API.  I have not tested wishlists because they are not used and I don't know how.

Reply:
And really, subtraction is the better way to do it. If you invert the bits, all higher order bits will be set, and the value will not actually equal all lower order flags or'd together.

Reply:
Nope. This is fine. I was just curious

Reply:
> higher order bits will be set

True